DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/AdminUserDataServiceTests.cs

```diff
@@ -193,5 +194,24 @@ public void UpdateAdminUserFailedLoginCount_updates_user()
                 transaction.Dispose();
             }
         }
+
+        [Test]
+        public void DeactivateAdminUser_updates_user()
+        {
+            using var transaction = new TransactionScope();
+            try
+            {
+                // When
+                userDataService.DeactivateAdmin(7);
+                var updatedUser = userDataService.GetAdminUserById(7)!;
+
+                // Then
+                updatedUser.Active.Should().Be(false);

```

Comment:
Active will always be false as GetAdminUserById does not populate Active, and therefore it defaults to false. Therefore this unit test does not actually test anything.

Reply:
Active now will populate from the query